source/draw/DrawState.ooc

```diff
@@ -28,6 +28,8 @@ DrawState: cover {
 	opacity := 1.0f
 	blendMode := BlendMode Fill
 	interpolate := true
+	flipSourceX := false
+	flipSourceY := false
```

Comment:
This new feature allow mirroring the source region for images that are in another coordinate system.